client/src/components/manageProjects/editMeetingTimes.js

```diff
@@ -63,50 +60,23 @@ const EditMeetingTimes = ({
         updatedDate,
       };
 
-      // If the day has been changed, find the next occurence of the changed day
-      if (values.day) {
-        const date = findNextOccuranceOfDay(values.day);
-        const dateGMT = new Date(date).toISOString();
-
-        theUpdatedEvent = {
-          ...theUpdatedEvent,
-          date: dateGMT,
-        };
-      }
-
-      // Set start time, End time and Duration if either start time or duration is changed
-      if (values.startTime || values.duration) {
-        /*
-        We need a start time and a duration to calculate everything we need.  
-        If only the start time or only the duration is changing, 
-        we use the previous time or duration for the calculation.
-        */
+      // Find next occurance of Day in the future
+      // Assign new start time and end time
```

Comment:
In `createMeetingTimes.js` the timing selections for a meeting are used to create a correct meeting time. We have access to the same information and it has been consistently correct, this is the most correct information we have about the meeting times.